src/main/webapp/js/userMap.js

```diff
@@ -0,0 +1,104 @@
+document.addEventListener("DOMContentLoaded", function(event) {
+    // based on example from https://github.com/markmarkoh/datamaps/blob/master/src/examples/highmaps_world.html
+    // Country code: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3
+
+    var countriesObj = {};
+    var countriesArr = [];
```

Comment:
Not entirely sure, but do we really need both of them? Can we process the data directly to `countriesArr` (which can then be renamed)?